protocol/x/clob/ante/clob.go

```diff
@@ -63,20 +63,8 @@ func (cd ClobDecorator) AnteHandle(
 	msgs := tx.GetMsgs()
 	var msg = msgs[0]
 
-	// Set request-level logging tags
-	ctx = log.AddPersistentTagsToLogger(ctx,
-		log.Module, log.Clob,
-		log.Callback, lib.TxMode(ctx),
```

Comment:
@jonfung-dydx Lmk if there are any specific logs that need Msg. Don't want to serialize it for every CheckTx.

Reply:
leave the Module=Clob tag

Reply:
Leaving that out for now to avoid the hundreds or thousands of allocs per second.